kbase-extension/static/kbase/js/common/jobManager.js

```diff
@@ -792,22 +783,18 @@ define([
                 });
 
                 Jobs.populateModelFromJobArray(this.model, Object.values(allJobs));
-
-                this._initJobs({ allJobIds, batchId: batch_id });
-
-                // request job info
-                this.addListener(jcm.MESSAGE_TYPE.INFO, jcm.CHANNEL.JOB, allJobIds);
-                this.bus.emit(jcm.MESSAGE_TYPE.INFO, { [jcm.PARAM.BATCH_ID]: batch_id });
+                this._initJobs({ batchId: batch_id });
             }
 
             _initJobs(args) {
-                const { allJobIds, batchId } = args;
-
-                ['STATUS', 'ERROR'].forEach((msgType) => {
-                    this.addListener(jcm.MESSAGE_TYPE[msgType], jcm.CHANNEL.JOB, allJobIds);
+                const { batchId } = args;
+                ['ERROR', 'INFO', 'LOGS', 'RETRY', 'STATUS'].forEach((msgType) => {
```

Comment:
Add listeners for all message types on the batch channel, rather than having many individual listeners on job channels.